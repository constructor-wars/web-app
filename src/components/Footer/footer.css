.footer_container {
  background-color: var(--dark);
  margin: auto;
  max-height: 100px;
  width: 100%;
  padding: 0 20px;
  position: absolute;
  bottom: 0;
}
