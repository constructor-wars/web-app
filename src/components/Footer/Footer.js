import React from 'react';
import './footer.css';

export default function Footer() {
  return (
    <div className="footer_container">
      <span>­ЪЄГ­ЪЄ░ ­ЪЄх­ЪЄ▒ заЂДзаЂбзаЂЦзаЂ«заЂДзаЂ┐­ЪЄг­ЪЄД заЂДзаЂбзаЂЦзаЂ«заЂДзаЂ┐­ЪЄ▒­ЪЄ╣ ­ЪЄф­ЪЄИзаЂДзаЂбзаЂЦ </span>
    </div>
  );
}
