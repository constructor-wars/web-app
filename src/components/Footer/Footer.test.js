import React from "react";
import Footer from "./Footer";
import renderer from "react-test-renderer";

describe("Footer", () => {
  it("matches the snapshot", () => {
    const tree = renderer.create(<Footer />).toJSON();
    expect(tree).toMatchInlineSnapshot(`
<div
  className="footer_container"
>
  <span>
    ­ЪЄГ­ЪЄ░ ­ЪЄх­ЪЄ▒ заЂДзаЂбзаЂЦзаЂ«заЂДзаЂ┐­ЪЄг­ЪЄД заЂДзаЂбзаЂЦзаЂ«заЂДзаЂ┐­ЪЄ▒­ЪЄ╣ ­ЪЄф­ЪЄИзаЂДзаЂбзаЂЦ 
  </span>
</div>
`);
  });
});
