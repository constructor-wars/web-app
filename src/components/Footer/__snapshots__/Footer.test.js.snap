// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer matches the snapshot 1`] = `
<div
  className="footer_container"
>
  <span>
    ­ЪЄГ­ЪЄ░ ­ЪЄх­ЪЄ▒ заЂДзаЂбзаЂЦзаЂ«заЂДзаЂ┐­ЪЄг­ЪЄД заЂДзаЂбзаЂЦзаЂ«заЂДзаЂ┐­ЪЄ▒­ЪЄ╣ ­ЪЄф­ЪЄИзаЂДзаЂбзаЂЦ 
  </span>
</div>
`;
